diff --git a/allianceauth/authentication/views.py b/allianceauth/authentication/views.py
index c5c036bf..f1e99987 100644
--- a/allianceauth/authentication/views.py
+++ b/allianceauth/authentication/views.py
@@ -172,7 +172,7 @@ def sso_login(request, token):
             # Go to Step 2
             return redirect('registration_register')
     # Logging in with an alt is not allowed due to security concerns.
-    token.delete()
+#    token.delete()
     messages.error(
         request,
         _('Unable to authenticate as the selected character. '
