diff --git a/allianceauth/authentication/views.py b/allianceauth/authentication/views.py
index c5c036bf..a88a5b5a 100644
--- a/allianceauth/authentication/views.py
+++ b/allianceauth/authentication/views.py
@@ -167,10 +167,10 @@ def sso_login(request, token):
             login(request, user)
             return redirect(request.POST.get('next', request.GET.get('next', 'authentication:dashboard')))
         elif not user.email:
-            # Store the new user PK in the session to enable us to identify the registering user in Step 2
-            request.session['registration_uid'] = user.pk
-            # Go to Step 2
-            return redirect('registration_register')
+            user.is_active = True
+            user.email = "fake-email@fnt.services"
+            user.save()
+            login(request, user, 'allianceauth.authentication.backends.StateBackend')
     # Logging in with an alt is not allowed due to security concerns.
     token.delete()
     messages.error(
