diff --git a/corptools/task_helpers/char_tasks.py b/corptools/task_helpers/char_tasks.py
index 28acfaf..bf73025 100644
--- a/corptools/task_helpers/char_tasks.py
+++ b/corptools/task_helpers/char_tasks.py
@@ -10,6 +10,7 @@ from django.utils import timezone
 from esi.models import Token
 
 from corptools.task_helpers.update_tasks import fetch_location_name
+from jsonschema.exceptions import ValidationError
 
 from .. import providers
 from ..models import (CharacterAsset, CharacterAudit, CharacterContact,
@@ -1087,6 +1088,10 @@ def update_character_notifications(character_id, force_refresh=False):
             audit_char.character.character_name))
         pass
 
+    except ValidationError as e:
+        logger.error(f"CT: Failed to parse notifications response {e.message}")
+        pass
+
     audit_char.last_update_notif = timezone.now()
     audit_char.save()
     audit_char.is_active()
