diff --git a/discordnotify/app_settings.py b/discordnotify/app_settings.py
index a48e32b..aee2de6 100644
--- a/discordnotify/app_settings.py
+++ b/discordnotify/app_settings.py
@@ -2,6 +2,11 @@
 
 from django.conf import settings
 
+DISCORDNOTIFY_DISCORDPROXY_HOST = getattr(
+    settings, "DISCORDNOTIFY_DISCORDPROXY_HOST", "localhost"
+)
+"""Host used to communicate with Discord Proxy."""
+
 DISCORDNOTIFY_DISCORDPROXY_PORT = getattr(
     settings, "DISCORDNOTIFY_DISCORDPROXY_PORT", 50051
 )
diff --git a/discordnotify/core.py b/discordnotify/core.py
index 4dd49bc..4b0ff46 100644
--- a/discordnotify/core.py
+++ b/discordnotify/core.py
@@ -7,7 +7,7 @@ from allianceauth.notifications.models import Notification
 from app_utils.urls import reverse_absolute, static_file_absolute_url
 
 from . import __title__
-from .app_settings import DISCORDNOTIFY_DISCORDPROXY_PORT, DISCORDNOTIFY_MARK_AS_VIEWED
+from .app_settings import DISCORDNOTIFY_DISCORDPROXY_HOST, DISCORDNOTIFY_DISCORDPROXY_PORT, DISCORDNOTIFY_MARK_AS_VIEWED
 
 # embed colors
 COLOR_INFO = 0x5BC0DE
@@ -44,7 +44,7 @@ def forward_notification_to_discord(
         level=level,
         timestamp=timestamp,
     )
-    target = f"localhost:{DISCORDNOTIFY_DISCORDPROXY_PORT}"
+    target = f"{DISCORDNOTIFY_DISCORDPROXY_HOST}:{DISCORDNOTIFY_DISCORDPROXY_PORT}"
     client = DiscordClient(target=target, timeout=DISCORD_PROXY_TIMEOUT)
     client.create_direct_message(user_id=discord_uid, embed=embed)
     _mark_as_viewed(notification_id)
